commerce_coordinator/apps/commercetools/tasks.py

```diff
@@ -60,17 +62,17 @@ def refund_from_stripe_task(
     try:
         payment = client.get_payment_by_key(payment_intent_id)
         if has_full_refund_transaction(payment):
-            logger.info(f"Stripe charge.refunded event received, but Payment with ID {payment.id} "
+            logger.info(f"[refund_from_stripe_task] event received, but Payment with ID {payment.id} "
                         f"already has a full refund. Skipping task to add refund transaction")
             return None
         updated_payment = client.create_return_payment_transaction(
```

Comment:
Are we not adding any log to tell that we are initiating the refund like we are doing above?

Reply:
Added.